hi-ml-histopathology/src/histopathology/utils/report_utils.py

```diff
@@ -109,14 +109,14 @@ def collect_crossval_metrics(parent_run_id: str, download_dir: Path, aml_workspa
     metrics_json = download_dir / "aml_metrics.json"
     if not overwrite and metrics_json.is_file():
         print(f"AML metrics file already exists at {metrics_json}")
-        metrics_df = pd.read_json(metrics_json)
+        metrics_df = pd.read_json(metrics_json, lines=True)
```

Comment:
Is it ok that we don't specify `orient='records'` here?

Reply:
Good catch. It worked in my tiny test, but I guess it's safest to add it here. I wonder if that's why the tests are failing. we'll see.

Reply:
[This is now outdated as I've refactored the PR]